lua/lvim/core/nvimtree.lua

```diff
@@ -14,6 +14,12 @@ function M.config()
         "dashboard",
         "alpha",
       },
+      auto_reload_on_write = true,
+      hijack_unnamed_buffer_when_opening = false,
+      hijack_directories = {
```

Comment:
I can see that `project.nvim` has changed the requirements, and it doesn't seem to mention netrw anymore, https://github.com/ahmedkhalf/project.nvim/blob/cef52b8da07648b750d7f1e8fb93f12cb9482988/README.md. 

Can you test this please
```diff
diff --git a/lua/lvim/core/nvimtree.lua b/lua/lvim/core/nvimtree.lua
index 45dc950..54622f1 100644
--- a/lua/lvim/core/nvimtree.lua
+++ b/lua/lvim/core/nvimtree.lua
@@ -132,9 +132,10 @@ function M.setup()
   if lvim.builtin.project.active then
     lvim.builtin.nvimtree.respect_buf_cwd = 1
     lvim.builtin.nvimtree.setup.update_cwd = true
-    lvim.builtin.nvimtree.setup.disable_netrw = false
-    lvim.builtin.nvimtree.setup.hijack_netrw = false
-    vim.g.netrw_banner = false
+    lvim.builtin.nvimtree.setup.update_focused_file = {
+      enable = true,
+      update_cwd = true,
+    }
   end
```